### PR TITLE
Adapt to standards

### PR DESCRIPTION
**Pull request by cbouss**
Changes:
- New feedstock at versio. 0.2.0

https://github.com/conda-incubator/conda-auth/tree/0.2.0

### CODE REVIEW

File: abs.yaml [hunk 1]
```diff
@@ -0,0 +1 @@
+aggregate_check: false
```
Comment (danpetry) [inline abs.yaml:1]: Why's this required out of interest?

**Review by danpetry — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
@cbouss Could you please patch the version of the package https://github.com/conda-incubator/conda-auth/blob/0.2.0/pyproject.toml#L7? The logs say:
```
  Source in $SRC_DIR has version 0.1.0, which satisfies requirement conda-auth==0.1.0 from file://$SRC_DIR
  Removed conda-auth==0.1.0 from file://$SRC_DIR from build tracker '/tmp/pip-build-tracker-dgygpz8t'
```
But you build **v0.2.0**.

I reported the upstream too https://github.com/conda-incubator/conda-auth/issues/25

File: abs.yaml (hunk 1 above)
Comment (M-Waszkiewicz-Anaconda) [reply abs.yaml:1]: yes, have the same question, why ?

Comment (cbouss) [reply abs.yaml:1]: It's a new feedstock and not pinned yet to aggregate. I could have added a pin instead.

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**